chapters/ch46.md

```diff
@@ -1,51 +1,52 @@
 
 > This is called an "autorotation" maneuver. It allows helicopters to land even if their engine unexpectedly fails. Human pilots practice this maneuver as part of their training. Your goal is to use a learning algorithm to fly the helicopter through a trajectory ​*T* t​hat ends in a safe landing.
 
---> _replace THIS LINE by your translation for the above line_
+Đây được gọi là kĩ thuật "quay tự động". Nó cho phép trực thăng hạ cánh ngay cả khi động cơ bị ngắt ngoài dự kiến. Kĩ thuật bay này là một phần trong việc huấn luyện của người phi công. Nhiệm vụ của bạn là sử dụng một thuật toán học tập để lái chiếc trực thăng qua một quỹ đạo *T* và kết thúc với một pha hạ cánh an toàn.
 
 > To apply reinforcement learning, you have to develop a "Reward function" ​*R​*(.) that gives a score measuring how good each possible trajectory ​*T​* is. For example, if ​*T* ​results in the helicopter crashing, then perhaps the reward is ​*R(T)​* = -1,000—a huge negative reward. A trajectory ​*T​* resulting in a safe landing might result in a positive ​*R(T)* w​ ith the exact value depending on how smooth the landing was. The reward function ​*R*(​.) is typically chosen by hand to quantify how desirable different trajectories ​*T​* are. It has to trade off how bumpy the landing was, whether the helicopter landed in exactly the desired spot, how rough the ride down was for passengers, and so on. It is not easy to design good reward functions.
 
---> _replace THIS LINE by your translation for the above line_
+Để áp dụng học tăng cường, bạn phải phát triển một "hàm điểm thưởng" *R*(.) nó sẽ cho một điểm số để đo xem mỗi quỹ đạo *T* tốt như thế nào. Lấy ví dụ, nếu *T* kết thúc bằng việc trực thăng bị rơi, thì điểm thưởng có thể sẽ là *R(T)* = -1.000-một điểm thưởng âm rất lớn. Một quỹ đạo *T* kết thúc bằng việc trực thăng hạ cánh an toàn có thể sẽ cho *R(T)* dương với giá trị chính xác phụ thuộc vào việc hạ cánh êm ái như thế nào. Hàm điểm thưởng *R*(.) thường được chọn thủ công để định lượng mức độ mong muốn của những quỹ đạo *T* khác nhau. Nó phải đánh đổi giữa những đặc tính như mức độ xóc của cú hạ cánh, trực thăng có hạ cánh đúng vị trí mong muốn không, quá trình hạ độ cao có nhiều biến động đối với hành khách không, và vân vân. Thiết kế một hàm điểm thưởng tốt không hề dễ dàng.
 
 > Given a reward function ​*R(T)*, ​the job of the reinforcement learning algorithm is to control the helicopter so that it achieves max​<sub>*T*</sub>​*R(T)*. ​However, reinforcement learning algorithms make many approximations and may not succeed in achieving this maximization.
 
---> _replace THIS LINE by your translation for the above line_
+Cho một hàm điểm thưởng *R(T)*, công việc của thuật toán học tăng cường là để điều khiển trực thăng sao cho nó đạt được điểm thưởng cao nhất max<sub>*T*</sub>*R(T)*. Tuy nhiên, thuật toán học tăng cường có nhiều phép ước lượng và có thể sẽ không thành công trong việc tối ưu này.
```

Comment:
- Với ~~Cho~~ một hàm điểm thưởng *R(T)* **_cho trước_**, công việc của thuật toán học tăng cường